src/main/java/io/jenkins/plugins/synopsys/security/scan/factory/ScanParametersFactory.java

```diff
@@ -436,38 +441,38 @@ private static void prepareBlackDuckToolConfigurationParametersMap(
             polarisParametersMap.put(
-                    ApplicationConstants.BLACKDUCK_SEARCH_DEPTH_KEY, securityScan.getPolaris_sca_search_depth());
+                    ApplicationConstants.BLACKDUCK_SEARCH_DEPTH_KEY, freestyleScan.getPolaris_sca_search_depth());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sca_config_path())) {
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sca_config_path())) {
             polarisParametersMap.put(
-                    ApplicationConstants.BLACKDUCK_CONFIG_PATH_KEY, securityScan.getPolaris_sca_config_path());
+                    ApplicationConstants.BLACKDUCK_CONFIG_PATH_KEY, freestyleScan.getPolaris_sca_config_path());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sca_args())) {
-            polarisParametersMap.put(ApplicationConstants.BLACKDUCK_ARGS_KEY, securityScan.getPolaris_sca_args());
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sca_args())) {
+            polarisParametersMap.put(ApplicationConstants.BLACKDUCK_ARGS_KEY, freestyleScan.getPolaris_sca_args());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sast_build_command())) {
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sast_build_command())) {
             polarisParametersMap.put(
-                    ApplicationConstants.COVERITY_BUILD_COMMAND_KEY, securityScan.getPolaris_sast_build_command());
+                    ApplicationConstants.COVERITY_BUILD_COMMAND_KEY, freestyleScan.getPolaris_sast_build_command());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sast_clean_command())) {
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sast_clean_command())) {
             polarisParametersMap.put(
-                    ApplicationConstants.COVERITY_CLEAN_COMMAND_KEY, securityScan.getPolaris_sast_clean_command());
+                    ApplicationConstants.COVERITY_CLEAN_COMMAND_KEY, freestyleScan.getPolaris_sast_clean_command());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sast_config_path())) {
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sast_config_path())) {
             polarisParametersMap.put(
-                    ApplicationConstants.COVERITY_CONFIG_PATH_KEY, securityScan.getPolaris_sast_config_path());
+                    ApplicationConstants.COVERITY_CONFIG_PATH_KEY, freestyleScan.getPolaris_sast_config_path());
         }
 
-        if (!Utility.isStringNullOrBlank(securityScan.getPolaris_sast_args())) {
-            polarisParametersMap.put(ApplicationConstants.COVERITY_ARGS_KEY, securityScan.getPolaris_sast_args());
+        if (!Utility.isStringNullOrBlank(freestyleScan.getPolaris_sast_args())) {
+            polarisParametersMap.put(ApplicationConstants.COVERITY_ARGS_KEY, freestyleScan.getPolaris_sast_args());
         }
```

Comment:
This method is not covered in unit tests. Is it possible to add some?

Reply:
Unit tests for this method are not covered during the arbitrary feature implementation PR. Maybe we can add them in our next feature implementation.

Reply:
You can add the tests in PR #147